es-ds-docs/pages/molecules/button.vue

```diff
@@ -0,0 +1,532 @@
+                                {{ example.text }}
+                                <temp-icon-chevron-right
+                                    v-if="example.icon"
+                                    :class="{ 'ml-25': example.text }" />
+                            </es-button>
+                        </template>
+                    </ds-responsive-table-column>
+                    <ds-responsive-table-column
+                        md="4"
+                        lg="3"
+                        xxl="2">
+                        <template #name>
+                            Disabled
+                        </template>
+                        <template #value>
+                            <es-button
+                                disabled
+                                outline
+                                :size="example.size"
+                                variant="dark-bg">
+                                {{ example.text }}
+                                <temp-icon-chevron-right
+                                    v-if="example.icon"
+                                    :class="{ 'ml-25': example.text }" />
+                            </es-button>
+                        </template>
+                    </ds-responsive-table-column>
+                </ds-responsive-table-row>
+            </ds-responsive-table>
+        </div>
+
+        <div class="my-500">
+            <h2>
+                Loading
+            </h2>
+            <p>
+                This example shows how to use the
+                <a
+                    href="https://bootstrap-vue.org/docs/components/spinner"
+                    target="_blank">
+                    spinner from Bootstrap Vue
+                </a>
+                to display a loading indicator within a button.
+            </p>
+            <div class="mb-100">
+                <es-button class="mr-100 position-relative">
+                    <span class="invisible">
+                        Default button
+                    </span>
+                    <temp-icon-spinner class="position-absolute" />
+                </es-button>
+                <es-button>
+                    <span>
+                        Default button
+                    </span>
+                    <temp-icon-spinner />
+                </es-button>
+            </div>
+            <div>
+                <es-button
+                    class="mr-100 position-relative"
+                    size="sm">
+                    <span class="invisible">
+                        Small button
+                    </span>
+                    <temp-icon-spinner class="position-absolute" />
+                </es-button>
+                <es-button size="sm">
+                    <span>
+                        Small button
+                    </span>
+                    <temp-icon-spinner />
+                </es-button>
+            </div>
+        </div>
+    </div>
```

Comment:
At the bottom of this page we do need to show the component source and docs source as is done on other docs pages. See the breadcrumbs docs for example